integration_tests/utils.py

```diff
@@ -410,8 +410,8 @@ def transfer(self, string):
         return receipt
 
 
-class TestRevert(Contract):
-    "TestRevert contract."
+class RevertTestContract(Contract):
```

Comment:
This rename is to remove a warning of pytest, because pytest will try to treat it as a test case.